packages/cactus-core-api/src/main/proto/weaver/common/protos/README.md

```diff
@@ -0,0 +1,8 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
```

Comment:
@petermetz 
is this line about copyright in line with the project's policy?

Reply:
@izuru0 @petermetz I discussed this with my team, and the copyright needs to be retained. Thanks!

Reply:
@izuru0 I think it is. My original intuition 2 years ago was also that maybe it is not, but then we had the same discussion about the copy right statements of Fujitsu [1] back then and it turned out that it was fine. I'm not a lawyer though.

[1] https://github.com/hyperledger/cactus/blob/0ae4ddb371febcb479d7749a19d98c14ec710d82/contribs/Fujitsu-ConnectionChain/connection-chain/environment/base/cc_env/servers/cooperation/ecSide/ec2_connector/lib/dependent/ServerPlugin.js#L2

Reply:
thanks for your comment and I think it is ok then

Reply:
@izuru0 @VRamakrishna  Thank you both, I'll get on with the merging then.